src/config/env.ts

```diff
@@ -0,0 +1,4 @@
+export const NEXT_PUBLIC_CONF_TOKEN_ADDRESS = process.env
+  .NEXT_PUBLIC_CONF_TOKEN_ADDRESS as `0x${string}`;
+export const NEXT_PUBLIC_PROJECT_ID = process.env
+  .NEXT_PUBLIC_PROJECT_ID as string;
```

Comment:
What about the other .env vars, should they be added too?
```suggestion
  .NEXT_PUBLIC_PROJECT_ID as string;
export const NEXT_PUBLIC_ACL_ADDRESS =  process.env
  .NEXT_PUBLIC_ACL_ADDRESS as `0x${string}`;
export const NEXT_PUBLIC_KMS_ADDRESS =  process.env
  .NEXT_PUBLIC_KMS_ADDRESS as `0x${string}`;
export const NEXT_PUBLIC_GATEWAY_URL =  process.env
  .NEXT_PUBLIC_GATEWAY_URL as string;
```